### DISCUSSION TITLE
how Livewire including frontend assets automatically?

### DISCUSSION THREAD

**Discussion by salahhusa9** [livewire/livewire#8309] [1. Help] [answered]
how Livewire including frontend assets automatically?

**Comment by salahhusa9** [comment]
@PhiloNL

**Comment by PhiloNL** [comment, marked as answer]
If it detects a Livewire component it will inject the assets: https://github.com/livewire/livewire/blob/main/src/Features/SupportAutoInjectedAssets/SupportAutoInjectedAssets.php
